test/integration/ProjectBrowserTest.kt

```diff
@@ -34,13 +37,13 @@ class ProjectBrowserTest: IntegrationTest() {
         studioTest(composeRule) {
             val createdDirectoryName = "created"
 
-            cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
+            cloneAndOpenProject(composeRule, source = Utils.SAMPLE_DATA_PATH, destination = testID)
 
             StudioState.project.current!!.directory.asDirectory().tryCreateDirectory(createdDirectoryName)
-            wait(composeRule, 500)
+            wait(composeRule, 750)
```

Comment:
I think we can extract this specific delay (was 500, now 750) as a particular type of delay. How about introducing a constant named `FILE_IO_DELAY`? (there may be a better name)

We can introduce other constants for other kinds of delays, like `CONNECT_SERVER_DELAY`.

Actually, if we have several of them, I'd propose creating an object named `Delays` in `Utils`, and referencing them as such: `Delays.FILE_IO`, `Delays.CONNECT_SERVER`. (Why not just `FILE_IO`? Because then it's not clear what the type is. Why not `FILE_IO_DELAY`? Because then we're repeating the word `DELAY` in our definitions unnecessarily)

Reply:
Done.